app/src/main/res/values/strings.xml

```diff
@@ -858,5 +858,6 @@
     GND. GND is meant for Ground and you can use any of the PSLab device GND pins as they are common.\n\n
     \u2022 Then go to the Configure tab from the bottom navigation bar and choose the BH-1750 in the drop down
     menu Select Sensor.\n</string>
+    <string name="device_not_connected_warning">PSLab device not connected</string>
```

Comment:
Similar string is available in 138-142 in `strings.xml`. Please reuse that one without creating new strings

Reply:
ok